src/test/java/com/integrivideo/stories/BasicChatStoryTest.java

```diff
@@ -124,5 +124,14 @@ public void demoComponentShouldHaveRestriction(){
         chatSteps.isThisIsTrialVersionModalShown();
     }
 
-    //TODO add test to validate message with url + click
+    @Test
+    public void sendMessageWithUrl() {
+        chatSteps.openDemoChat();
+        chatSteps.sendWithEnter(Data.MSG_WITH_URL);
+        chatSteps.messageTextShouldBeLike(1, Data.MSG_WITH_URL_FORMATTED);
```

Comment:
Incorrect expected result. Variable should be the same as in this line chatSteps.sendWithEnter(Data.MSG_WITH_URL);